app/_src/guides/federate-kv.md

```diff
@@ -0,0 +1,181 @@
+
+```sh
+export KDS_IP=host.minikube.internal
+```
+{% endtip %}
+
+## Copy resources from zone to global control plane
+
+To federate zone control plane without any traffic interruption, we need to copy resources like secrets, meshes etc.
+First, we need to expose API server of zone control plane:
+
+```sh
+kubectl --context=mesh-zone port-forward svc/{{site.mesh_cp_name}} -n {{site.mesh_namespace}} 5681:5681
+```
+
+Then we export resources:
+```sh
+export ZONE_USER_ADMIN_TOKEN=$(kubectl --context=mesh-zone get secrets -n {{site.mesh_namespace}} admin-user-token -ojson | jq -r .data.value | base64 -d)
+kumactl config control-planes add \
+  --address http://localhost:5681 \
+  --headers "authorization=Bearer $ZONE_USER_ADMIN_TOKEN" \
+  --name "zone-cp" \
+  --overwrite  
+  
+kumactl export --profile=federation-with-policies --format=kubernetes > resources.yaml
+```
+
+And finally, we apply resources on global control plane
+```sh
+kubectl apply --context=mesh-global -f resources.yaml
+```
+
+## Connect zone control plane to global control plane
+
+Update Helm deployment of zone control plane to configure connection to the global control plane.
+
+
+```sh
+helm upgrade --kube-context=mesh-zone --namespace {{site.mesh_namespace}} \
+--set {{site.set_flag_values_prefix}}controlPlane.mode=zone \
+--set {{site.set_flag_values_prefix}}controlPlane.zone=zone-1 \
+--set {{site.set_flag_values_prefix}}ingress.enabled=true \
+--set {{site.set_flag_values_prefix}}controlPlane.kdsGlobalAddress=grpcs://${KDS_IP}:5685 \
+--set {{site.set_flag_values_prefix}}controlPlane.tls.kdsZoneClient.skipVerify=true \
+{{ site.mesh_helm_install_name }} {{ site.mesh_helm_repo }}
+```
+
+## Verify federation
+
+To verify federation, first port-forward the API service from the global control plane to port 15681 to avoid collision with previous port forward. 
+
+```sh
+kubectl --context=mesh-global port-forward svc/{{site.mesh_cp_name}} -n {{site.mesh_namespace}} 15681:5681
+```
+
+And then navigate to [127.0.0.1:15681/gui](http://127.0.0.1:15681/gui) to see the GUI.
+
+You should eventually see
+* a zone in list of zones
+* policies including `redis` MeshTrafficPermission that we applied in the quickstart guide.
+* data plane proxies for the demo application that we installed in the quickstart guide.
+
+### Apply policy on global control plane
+
+We can check policy synchronization from global control plane to zone control plane by applying a policy on global control plane:
+
+```sh
+echo "apiVersion: kuma.io/v1alpha1
+kind: MeshCircuitBreaker
+metadata:
+  name: demo-app-to-redis
+  namespace: kuma-demo
+  labels:
+    kuma.io/mesh: default
+spec:
+  targetRef:
+    kind: Dataplane
+    labels:
+      app: demo-app
+  to:
+  - targetRef:
+      kind: MeshService
+      name: redis
+    default:
+      connectionLimits:
+        maxConnections: 2
+        maxPendingRequests: 8
+        maxRetries: 2
+        maxRequests: 2" | kubectl --context=mesh-global apply -f -
```

Comment:
Let's be consistent and not use `=` after the flag names whenever it's unnecessary
```suggestion
        maxRequests: 2" | kubectl --context mesh-global apply -f -
```